include/spirv-tools/libspirv.h

```diff
@@ -53,6 +53,7 @@ typedef enum spv_result_t {
   SPV_ERROR_INVALID_CFG = -11,
   SPV_ERROR_INVALID_LAYOUT = -12,
   SPV_ERROR_INVALID_CAPABILITY = -13,
+  SPV_ERROR_INVALID_DATA = -14,
```

Comment:
Do you mean documentation inside the file (as comments), or do we have a document (.doc) that I need to update?